presto-main/src/main/java/com/facebook/presto/operator/SetBuilderOperator.java

```diff
@@ -189,12 +193,30 @@ public boolean isFinished()
         return finished;
     }
 
+    /**
+     * This function processes unfinished work for the current page, and returns if it still
+     * needs input from the previous(child) operator.
+     * @return true if this operator needs additional input pages from its child, false otherwise
```

Comment:
Remove this. Comment inside the method is definitely sufficient

Reply:
ok